### DISCUSSION TITLE
Avo css overriding my other app styles

### DISCUSSION THREAD

**Discussion by mwala-zm** [avo-hq/avo#3552] [Help] [answered]
I have an app to which I want to add avo as an admin dashboard. once I run `bin/rails app:template LOCATION='https://avohq.io/app-template'` and everything is generated avo overrides my css. My app is using tailwind to be specific.
![image](https://github.com/user-attachments/assets/cc56f381-eb1a-4b12-88fd-8e953ff2ab3e)
This pushes all my content to the bottom. Does anyone know if this is a bug or something I have to configure to get the css working separately.

**Comment by adrianthedev** [comment]
Hey @mwala-zm

That shouldn't happen AFAIK. 
Do you have a screenshot of the git diff?
I am away right now. I'll have a look as soon as I can. 

One thing you should ensure is that you're not loading Avo's css in your application.html file.

**Reply by mwala-zm** [reply]
@adrianthedev These are the only diffs I have at the moment
![image](https://github.com/user-attachments/assets/0814e941-5dea-4a23-b36b-bacac2783f3b)
![image](https://github.com/user-attachments/assets/b376a9d1-1acf-4bd2-bded-dd7e4276af1a)
![image](https://github.com/user-attachments/assets/dc934e4c-b826-4f59-bb79-de5d6d751d47)

**Reply by adrianthedev** [reply]
There's nothing there that would break your styles. 
I'd check to see if you're importing or injecting styles somehow. 

Is this a new or existing app?
Where are the styles overriden? Inside Avo or on the public site?

**Reply by mwala-zm** [reply]
It is an existing app
the styles where overridden on the public site. Avo is just working fine

**Reply by mwala-zm** [reply, marked as answer]
So avo was conflicting with daisyui and that caused the app's css to bug out. adding some classes to the html tag fixed it.
